src/app/CommandSender.cpp

```diff
@@ -441,6 +441,14 @@ CHIP_ERROR CommandSender::ProcessInvokeResponseIB(InvokeResponseIB::Parser & aIn
             }
         }
 
+        if (!commandRef.HasValue() && !commandRefRequired && mpPendingResponseTracker != nullptr && mpPendingResponseTracker->Count() == 1)
```

Comment:
Please add a test to cover detecting a regression of this PR's fix.

Reply:
Test has been added